abstract.tex

```diff
@@ -1,7 +1,8 @@
 \begin{abstract}
         The French 2012-2015 \textit{Commission Nationale d'Evaluation Reports}
 emphasize preparation for a transition from \glspl{LWR} to \glspl{SFR}.
-We used the \Cyclus nuclear fuel cycle simulator to explore the feasibility of enabling a French
+We used the \Cyclus nuclear fuel cycle simulator to explore the feasibility of 
+        boostrapping a French
```

Comment:
I personally think the term bootstrap is confusing, but it probably comes from my unfamiliarity with the term

Reply:
It comes from "pulling oneself up by ones bootstraps." Anyway, to your point, as it's an international journal, I probably shouldn't use such  US-specific jargon. Fixing (here and in letter).